plugins/outputs/cloudwatchlogs/internal/pusher/sender.go

```diff
@@ -134,6 +138,14 @@ func (s *sender) Send(batch *logEventBatch) {
 	}
 }
 
+func (s *sender) Stop() {
+	if s.stopped {
+		return
+	}
+	close(s.stopCh)
+	s.stopped = true
+}
```

Comment:
Not thread-safe, but likely not an issue since the sender should only be stopped from a single thread.